recipes-wpe/wpeframework/include/power.inc

```diff
@@ -4,12 +4,14 @@ WPE_POWER_AUTOSTART ?= "true"
 WPE_POWER_GPIOPIN ?= ""
 WPE_POWER_GPIOTYPE ?= ""
 WPE_POWER_DEP = "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/egl', 'broadcom-refsw', 'broadcom-refsw', '', d)}"
+POWER_PLATFORM_IMPL ?= "Nexus"
```

Comment:
Can we select the stub incase we're not building for Broadcom? And please prefix it with WPE_POWER_

i.e.
```
WPE_POWER_PLATFORM_IMPL ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/egl', 'broadcom-refsw', 'Nexus', 'stub', d)}"
```

This will build the stub impl if someone accidentally it for a platform that isn't supported OR we can enable power by default for all builds.

Reply:
Updated as suggested.